src/StackAdmin/Azs.Subscriptions.Admin/readme.md

```diff
@@ -217,7 +223,7 @@ directive:
       parameter-name: State
     set:
       default:
-        script: 'Private'
+        script: Echo "Private"
```

Comment:
can you please add a comment ? why echo?

Reply:
need this parameter value to be a string. just tried now, other option is '"Private"'.
If the default script is set to 'Private', it is trying to invoke Private and throwing Private cmd not found error.